src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1777,10 +1778,13 @@ export const ComputeModalBase = ({
       ])
     ])
 
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const idDiskType = useUniqueId()
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const idDiskSize = useUniqueId()
```

Comment:
To minimize the scope, I would suggest you create `idDiskSize` within `renderPersistentDiskType`. There's really no reason to have to pass it in.

Reply:
➕